src/compiler/sourcemap.ts

```diff
@@ -333,12 +336,21 @@ namespace ts {
                 const oldSource = currentSource;
                 if (source === oldSource) source = undefined;
 
+                let nameIndex;
+                if (isIdentifier(node)) {
+                    const originalName = getTextOfNodeFromSourceText(currentSourceText, node);

```

Comment:
Not every `Identifier` has original source text. In those cases, `getTextOfNodeFromSourceText` would return `""`, which isn't a valid name. For those cases, we shouldn't define a name index.

Reply:
fixed